packages/legacy/core/cli.js

```diff
@@ -36,7 +36,16 @@ function mapPackages(jsonFile, section, aggregated) {
   }
 }
 
-function syncPackageJson(sourcePackageJsonFilePath, destPackageJsonFilePath) {
+/**
+ *
+ * @param {Object} opts
```

Comment:
Not overly helpful documentation.

Reply:
True, sorry. Part of some copied lines from line 150 - line 160